.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy extension
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish-marketplace:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Deploy to marketplace
+        run: npm run vscode:publish
+        env:
+          VSCE_PAT: ${{ secrets.VSCODE_SECRET }}
```

Comment:
I guess we need to set this up in the repository settings - Some note about this would be nice in the description.